### PR TITLE
Fixing Popen in JJB

### PR DESCRIPTION
**Pull request by mkluson**
Adding parameter stdout=subprocess.PIPE to Popen(), so the output is string now.
Determination of error in Popen call changed to simple check of empty output.

### CODE REVIEW

**Comment by optak** [comment]
I'm going to test this PR locally before merging into master.

**Comment by optak** [comment]
testing failed, do not merge this yet

**Comment by optak** [comment]
This patch is ok, my testing failed because of another bug in baseos-ci code.
I think it's safe to merge it now.

**Review by lulinqing — APPROVED** [review]

**Pull request merged**
